solutions/security/ai/ai-assistant.md

```diff
@@ -26,13 +26,12 @@ The Elastic AI Assistant is designed to enhance your analysis with smart dialogu
 ::::
 
 ::::{admonition} Requirements
-* {{stack}} users: {{stack}} version 8.8.1 or later. Also note the Generative AI connector was renamed to OpenAI connector in 8.11.0.
-* {{stack}} users: an [Enterprise subscription](https://www.elastic.co/pricing).
-* {{serverless-short}} users: a [Security Analytics Complete subscription](/deploy-manage/deploy/elastic-cloud/project-settings.md).
+* {applies_to}`stack: ga` An [Enterprise subscription](https://www.elastic.co/pricing).
+* {applies_to}`serverless: ga` A [Security Analytics Complete subscription](/deploy-manage/deploy/elastic-cloud/project-settings.md).
```

Comment:
```suggestion
* {applies_to}`serverless: ga` An {{sec-serverless}} project with the [Security Analytics Complete feature tier](/deploy-manage/deploy/elastic-cloud/project-settings.md).
```
This is a feature tier, not a subscription - found a couple of other occurrences https://github.com/elastic/docs-content/issues/2768

Reply:
The new EASE tier also supports assistant functionality.